opencti-platform/opencti-front/src/utils/edition.js

```diff
@@ -14,6 +14,9 @@ export const convertMarking = (element) => ({
   label: element.definition ?? element.id,
   value: element.id,
   color: element.x_opencti_color,
+  definition_type: element.definition_type,
+  x_opencti_order: element.x_opencti_order,
+  entity: element,
```

Comment:
Not sure to understand why we have this line

Reply:
Yes, entity can be removed now, it was used before some changes :)